index.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Persaonal Portfolio</title>
+    <link rel="stylesheet" href="styles.css" />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.2.1/css/all.min.css"
+      integrity="sha512-MV7K8+y+gLIBoVD59lQIYicR65iaqukzvf/nwasF0nqhPay5w/9lJmVM2hMDcnK1OnMGCdVK+iQrJ7lzPJQd1w=="
+      crossorigin="anonymous"
+      referrerpolicy="no-referrer"
+    />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Inter:wght@600&display=swap"
+      rel="stylesheet"
+    />
+  </head>
+
+  <body class="background">
+    <ul class="flex-container">
+      <li><h1 class="welcome">Welcome</h1></li>
+
+      <li><i class="fa-solid fa-bars fa-2x"></i></li>
+    </ul>
+    <header class="header-section">
+      <h2 class="header-section-title">
+        <p class="header-name">Hey there. I’m Coral</p>
+        <p class="header-name-header-title">I’m a software developer</p>
+      </h2>
+    </header>
+    <p class="paragraph">
+      I can help you build a product, feature or website Look through some of my
+      work and experience! If you like what you see and have a project you need
+      coded, don’t hestiate to contact me.
+    </p>
+    <ul class="icon">
+      <li><img src="twitter.jpg" width="19.99px" height="19.96px" /></li>
+      <li><img src="linkedin icon.jpg" width="20px" height="20px" /></li>
+      <li><img src="medium.jpg" width="20px" height="16px" /></li>
+      <li><img src="github.jpg" width="17.99px" height="20px" /></li>
+      <li><img src="angellist.jpg" width="14.17px" height="20px" /></li>
```

Comment:
- [ ] Great Job wrapping ur social link in a ul tag , now each icon should be an anchor link example : 
```HTML
<li><a href="#"><img src="twitter.jpg" width="19.99px" height="19.96px" /> </a</li>
```
kindly fix it 🙏